Adding transition sentences to the github actions extra (#104)
carpentries-incubator/docker-introduction: _extras/e01-github-actions.md

Only a `index.html` and `.nojekyll` (that prevents Github from creating a Jekyll page). So how do we
set this up?

## Create a Github Project
Create a github project with a short `README.md`. To do this:

- go to `github.com` and make sure you're logged in

> - click the green "New" button at the top right)

Reviewer: ```suggestion
- click the green "New" button at the top right
```

~~~
{: .output}

> Now, we can run pandoc on our README.md file by including our current directory and

Reviewer: ```suggestion
Now, we can run pandoc on our `README.md` file by including our current directory and
```

> the README.md file as part of the `docker run` command:

Reviewer: ```suggestion
the `README.md` file as part of the `docker run` command:
```

~~~
{: .output}

> Here, the `-v ${PWD}:/tmp` flag says to take the directory at `${PWD}` and expose it inside the

Reviewer: ```suggestion
Here, the `-v ${PWD}:/tmp` flag says to take the directory at `${PWD}` and make it available inside the
```
